### PR TITLE
fix: Invalid color settings passed to nvim_set_hl 

### PR DESCRIPTION
**Pull request by npmaile**
this is a fix to this issue:
https://github.com/hrsh7th/nvim-cmp/issues/961
Not sure if it's the correct fix, but from what we (the members of the @theprimeagen discord server @navigationhazard and @mvargasmoran) seem to think it's correct

### CODE REVIEW

**Comment by npmaile** [comment]
addresses https://github.com/hrsh7th/nvim-cmp/issues/961

File: lua/cmp/utils/highlight.lua [hunk 1]
```diff
@@ -20,7 +20,7 @@ highlight.inherit = function(name, source, settings)
         v = v == 1
       end
       if v then
-        settings[key] = v == '' and 'NONE' or v
+        settings[key] = v == '' or 'NONE' or '0'
```
Comment (hrsh7th) [inline lua/cmp/utils/highlight.lua:23]: `'NONE' or '0'` always return 'NONE'.

**Comment by hrsh7th** [comment]
Thank you.
I can merge this for now. But I think it still not corrext to me.

**Comment by hrsh7th** [comment]
Ah! Sorry. I'm getting the free wi-fi spot. I'll double check about this issue.

**Pull request closed without merging**